clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -139,7 +146,22 @@
     ILLEGAL_SASL_STATE(34,
             new IllegalSaslStateException("Request is not valid given the current SASL state.")),
     UNSUPPORTED_VERSION(35,
-            new UnsupportedVersionException("The version of API is not supported."));
+            new UnsupportedVersionException("The version of API is not supported.")),
+    TOPIC_ALREADY_EXISTS(36,
+            new TopicExistsException("Topic with this name already exists.")),
+    INVALID_PARTITIONS(37,
+            new InvalidPartitionsException("Number of partitions is invalid.")),
+    INVALID_REPLICATION_FACTOR(38,
+            new InvalidReplicationFactorException("Replication-factor is invalid.")),
+    INVALID_REPLICA_ASSIGNMENT(39,
+            new InvalidReplicaAssignmentException("Replica assignment is invalid.")),
+    INVALID_CONFIG(40,
+            new InvalidConfigurationException("Configuration is invalid.")),
+    NOT_CONTROLLER(41,
+        new NotControllerException("This is not the correct controller for this cluster.")),
+    INVALID_REQUEST(42,
+        new InvalidRequestException("This most likely occurs because of a request being malformed by the client library or" +
+            " the message was sent to an incompatible broker. See the broker logs for more details."));
```

Comment:
I will add it. 

Note: That since only a newer client (the admin client) can actually make a request that causes this error, it should never show up in the old scala clients.